src/main/scala/com/fulcrumgenomics/alignment/Alignment.scala

```diff
@@ -308,6 +310,43 @@ object Alignment {
   def apply(query: String, target: String, queryStart: Int, targetStart: Int, cigar: Cigar, score: Int): Alignment = {
     new Alignment(query.getBytes, target.getBytes, queryStart, targetStart, cigar, score)
   }
+
+  /** Constructs a Cigar from a padded alignment. */
+  def cigarFrom(queryPadded: String, targetPadded: String, padChar: Char = '-', softClipChar: Char = 'X', useEqualsAndX: Boolean = false): Cigar = {
```

Comment:
used in `IndelPaddedAlignment` but generally useful, so putting it in `Alignment`